docs/migration/v11.md

```diff
@@ -0,0 +1,1657 @@
+# v11
+
+**Note: everything in this file is a work-in-progress and can be changed.**
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+## Table of Contents
+
+- [Upgrade paths](#upgrade-paths)
+- [carbon-components](#carbon-components)
+- [carbon-components-react](#carbon-components-react)
+- [carbon-icons](#carbon-icons)
+- [@carbon/icons](#carbonicons)
+- [@carbon/icons-react](#carbonicons-react)
+- [@carbon/colors](#carboncolors)
+- [@carbon/elements](#carbonelements)
+- [@carbon/grid](#carbongrid)
+- [@carbon/layout](#carbonlayout)
+- [@carbon/motion](#carbonmotion)
+- [@carbon/pictograms](#carbonpictograms)
+- [@carbon/pictograms-react](#carbonpictograms-react)
+- [@carbon/themes](#carbonthemes)
+- [@carbon/type](#carbontype)
+- [FAQ](#faq)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+This document outlines all of the changes to the Carbon Design System from v10
+to v11. If you're just getting started, it may be helpful to checkout our
+[migration site](#todo).
```

Comment:
Is the "migration site" here the sample app or the v11.carbondesignsystem.com one?